cmd/crossbuild.go

```diff
@@ -136,8 +148,12 @@ func runCrossbuild() {
 	}
 
 	if MIPSPlatformsParam := strings.Join(MIPSPlatforms[:], " "); MIPSPlatformsParam != "" {
```

Comment:
`MIPS` is originally an acronym for `Microprocessor without Interlocked Pipeline Stages`, so I'm in favor to leave it in uppercase like ARM.